osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -119,7 +119,16 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
             logo.FadeIn(100, Easing.OutQuint);
 
             if (resuming)
+            {
                 buttons.State = MenuState.TopLevel;
+
+                const float length = 300;
+
+                Content.FadeIn(length, Easing.OutQuint);
+                Content.MoveTo(new Vector2(0, 0), length, Easing.OutQuint);
+
+                sideFlashes.Delay(length).FadeIn(64, Easing.InQuint);
+            }

```

Comment:
This stuff doesn't look related to the logo o_O, does this really need to be moved outside of OnResuming?

Reply:
it's the whole point of this PR. previously it was running before the logo was ready and therefore was out-of-sync.

Reply:
Ah, I see the effect of it now.